src/test/java/ibm/jceplus/junit/openjceplusfips/TestRSASignatureInteropNonFIPS.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright IBM Corp. 2023
+ *
+ * Licensed under the Apache License 2.0 (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution.
+ */
+package ibm.jceplus.junit.openjceplusfips;
+
+import java.security.KeyPair;
+import java.security.KeyPairGenerator;
+import java.security.PrivateKey;
+import java.security.PublicKey;
+import java.security.Signature;
+
+import junit.framework.Test;
+import junit.framework.TestSuite;
+
+public class TestRSASignatureInteropNonFIPS 
+    extends ibm.jceplus.junit.base.BaseTestRSASignatureInterop {
+
+    //--------------------------------------------------------------------------
+    //
+    //
+    static {
+        Utils.loadProviderTestSuite();
+    }
+
+    //--------------------------------------------------------------------------
+    //
+    //
+    public TestRSASignatureInteropNonFIPS() {
+        super(Utils.TEST_SUITE_PROVIDER_NAME, Utils.PROVIDER_SunRsaSign, 1024);
+    }
+
+    // RSA signature allows at least 2048 bits of RSA key to be used for sign a signature.
+    @Override
+    protected void doSignVerify(String sigAlgo, byte[] message, PrivateKey privateKey,
```

Comment:
I think we are missing a few interop cases here, as the method that is overridden was calling other methods that would use different providers for different things. In this case, `OpenJCEPlusFIPS` is used for both sign/verify, while `Sun` is used for key generation.

Reply:
Openjceplusfips does not allow a 1024 size of key pair, we have to use other providers such as SunRsaSign to create a 1024 keysize key. 

In this doSignVerify test, atm, we only test whether sign procedure still accepts 1024 key size for RSA or not.

Reply:
So, maybe we'll need a different name, as this is not `interop`. We would need a provider to sign and a different one to verify in order to be `interop`.

Reply:
Yes, we use the same provider to do the sign and verify, but a different provider to generate the key pair. 

Didnt come up with a good name for this test, any idea?

Reply:
I think it would be `TestRSASignatureVerifySmall` or something. Because it's still FIPS, but with an externally created key, right?

Reply:
Changed the name to `TestRSASignatureWithSpecificSize`